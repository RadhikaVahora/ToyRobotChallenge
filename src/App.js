import './App.css';
import TabsView from './Components/Tabs/TabsView';

function App() {
  return (
    <div className="App">
      <TabsView />
    </div>
  );
}

export default App;
