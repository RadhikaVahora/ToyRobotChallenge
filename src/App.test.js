
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import App from './App';

describe('AppView', () => {
  it('should render correctly',() =>{
    const renderer = new ShallowRenderer();
    const wrapper = renderer.render(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
