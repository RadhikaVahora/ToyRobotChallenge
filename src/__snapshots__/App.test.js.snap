// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppView should render correctly 1`] = `
<div
  className="App"
>
  <TabsView />
</div>
`;
